api/__init__.py

```diff
@@ -0,0 +1,54 @@
+import base64
+
+from flask import Flask, request
+import numpy as np
+
+
+def create_app():
+    app = Flask(__name__)
+
+    @app.route('/')
+    def routes():
+        return {
+            "spectrum": "/spectrum",
+            "mean": "/mean",
+            "median": "/median",
+        }
+
+    @app.route('/spectrum', methods=['POST', 'GET'])
+    def spectrum():
+        if request.method == 'GET':
```

Comment:
Why overload this endpoint with GET and POST.  Special logic is then needed to determine what type of request this is a implement different behavior based on the request type.

Would it not be cleaner to have two separate deceleration of the /spectrum endpoint and all end point below?

Reply:
Same for
- [ ] https://github.com/cwc-products/python-sample/pull/2/files#diff-387e0a038c4ca5f5cc74c8f060afc0a9R34 
- [ ] https://github.com/cwc-products/python-sample/pull/2/files#diff-387e0a038c4ca5f5cc74c8f060afc0a9R34

Reply:
#3